getting-started/nextjs.mdx

```diff
@@ -170,7 +175,7 @@ sure to keep your desired functionality in mind when choosing which components t
 		}
 
 		export const getServerSideProps: GetServerSideProps = async (context) => {
-			const user = await getUserFromServerSideProps(context)
+			const user = await getUserFromServerSideProps(context, true)
```

Comment:
You can leave this out. I think we'll actually be better off splitting up app router + pages router and then we can go into more details here.